.github/workflows/ci.yml

```diff
@@ -85,6 +85,10 @@ jobs:
           distribution: temurin
           java-version: '8'
 
+      - run: |
```

Comment:
`          if: ${{ matrix.name == 'Linux' }}`

Reply:
Yes was waiting for the workflow to continue.

Did you check compression?

Reply:
last one failed; waiting for latest.